libursa/Indexer.cpp

```diff
@@ -62,7 +62,7 @@ void Indexer::remove_dataset(const OnDiskDataset *dataset_ptr) {
 }
 
 void Indexer::make_spill(DatasetBuilder &builder) {
-    std::cout << "new dataset" << std::endl;
+    spdlog::debug("new dataset");
```

Comment:
Do you mind moving this one line lower and also logging `dataset_name`? Just in case